src/coreclr/inc/cordebuginfo.h

```diff
@@ -192,11 +192,11 @@ class ICorDebugInfo
 #elif TARGET_AMD64
         REGNUM_SP = REGNUM_RSP,
 #elif TARGET_ARM
-#ifdef REDHAWK
+#ifdef FEATURE_NATIVEAOT
```

Comment:
This file doesn't compile into NativeAOT. This is some very old define. I would delete it, I don't even know if it's true.

Refreshing the name from REDHAWK to NativeAOT would make it look like the information is fresh.